### PR TITLE
Add PR template before enabling automatic deploys

### PR DESCRIPTION
**Pull request by benthorner**
https://trello.com/c/epNWudCR/176-write-an-rfc-for-continuous-deployment-and-seek-review

This is part of enabling automatic deployments for this app.

:warning: This application is Continuously Deployed: :warning:

- Merged changes are automatically deployed to staging and production.

- Make sure you follow [the guidance for deployments](https://docs.publishing.service.gov.uk/manual/development-pipeline.html#merge-your-own-pull-request) **before** you merge.

- Check your branch is being deployed in the [Release app](https://release.publishing.service.gov.uk/applications/smartanswers), after merging.

- Changes to start pages are **not** continuously deployed, and are [deployed using a different process](https://github.com/alphagov/smart-answers/blob/master/doc/smart-answer-flow-development/publishing.md).

### CODE REVIEW

**Comment by benthorner** [comment]
Adding "DO NOT MERGE" until RFC 128 is concluded.

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+:warning: This application is Continuously Deployed: :warning:
+
+- Merged changes are automatically deployed to staging and production.
+
+- Make sure you follow [the guidance for deployments](https://docs.publishing.service.gov.uk/manual/development-pipeline.html#merge-your-own-pull-request) **before** you merge.
+
+- Check your branch is being deployed in the [Release app](https://release.publishing.service.gov.uk/applications/smartanswers), after merging.
```
Comment (thomasleese) [inline .github/pull_request_template.md:7]:
```suggestion
- Check your branch is being deployed in the [Release app](https://release.publishing.service.gov.uk/applications/info-frontend), after merging.
```

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+- Changes to start pages are **not** continuously deployed, and are [deployed using a different process](https://github.com/alphagov/smart-answers/blob/master/doc/smart-answer-flow-development/publishing.md).
```
Comment (thomasleese) [inline .github/pull_request_template.md:9]: I don't think this is relevant here.

File: .github/pull_request_template.md (hunk 1 above)
Comment (benthorner) [reply .github/pull_request_template.md]: Also fixed 👍.

File: .github/pull_request_template.md (hunk 2 above)
Comment (benthorner) [reply .github/pull_request_template.md]: Also removed 😇.

**Review by thomasleese — APPROVED** [review]

**Pull request merged**